examples/experimental/gltf-with-raw-webgl/GlbBuilder.js

```diff
@@ -0,0 +1,375 @@
+
+            if (uUseLighting) {
+
+                vec4 textureColor = texture(uTexture, vec2(vVertexTextureCoordinate.s, vVertexTextureCoordinate.t));
+                color = vec4(textureColor.rgb * vLightWeighting, textureColor.a);
+            }
+            else {
+
+                color = texture(uTexture, vec2(vVertexTextureCoordinate.s, vVertexTextureCoordinate.t));
+            }
+        }
+    `;
+
+    // endregion
+    // -----------------------------------------------------------------------------------------------------------------
+
+
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // region Attributes
+
+    gl = null;
+    mShaderProgram = null;
+    mShaderProgramInfo = null;
+
+    // endregion
+    // -----------------------------------------------------------------------------------------------------------------
+
+
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // region Constructor
+
+    constructor(gl) {
+
+        this.gl = gl;
+
+        this.mShaderProgram = Utils.initShaderProgram(gl, this.VERTEX_SHADER_SOURCE, this.FRAGMENT_SHADER_SOURCE);
+
+        this.mShaderProgramInfo = {
+
+            attribLocations: {
+                vertexPosition: this.gl.getAttribLocation(this.mShaderProgram, 'aVertexPosition'),
+                vertexNormal: this.gl.getAttribLocation(this.mShaderProgram, 'aVertexNormal'),
+                vertexTextureCoordinate: this.gl.getAttribLocation(this.mShaderProgram, 'aVertexTextureCoordinate'),
+            },
+
+            uniformLocations: {
+
+                projectionMatrix: this.gl.getUniformLocation(this.mShaderProgram, 'uProjectionMatrix'),
+                modelViewMatrix: this.gl.getUniformLocation(this.mShaderProgram, 'uModelViewMatrix'),
+                normalMatrix: this.gl.getUniformLocation(this.mShaderProgram, 'uNormalMatrix'),
+
+                useLightning: this.gl.getUniformLocation(this.mShaderProgram, 'uUseLighting'),
+
+                ambientColor: this.gl.getUniformLocation(this.mShaderProgram, 'uAmbientColor'),
+                lightingDirection: this.gl.getUniformLocation(this.mShaderProgram, 'uLightingDirection'),
+                directionalColor: this.gl.getUniformLocation(this.mShaderProgram, 'uDirectionalColor'),
+
+                texture: this.gl.getUniformLocation(this.mShaderProgram, 'uTexture')
+            }
+        };
+    }
+
+    // endregion
+    // -----------------------------------------------------------------------------------------------------------------
+
+
+
+    // -----------------------------------------------------------------------------------------------------------------
+    // region Loading
+
+    async load(glbUrl) {
+
+        // Fetch GLB file:
+        const glb = await Loaders.load(glbUrl, Loaders.GLBLoader, { gltf: { postProcess: false } });
```

Comment:
The loaders.gl `GLBLoader` doesn't have a post processing option? Is this your own bundled loader?

I assume we ultimately want to use the loaders.gl `GLTFLoader` as it loads all the linked resources?

Reply:
Indeed, that's a remaining from a copy paste using the GLTFLoader, I thought the GLB had the same options.